src/plugins/remark/default-code-snippet-lang/README.md

```diff
@@ -0,0 +1,7 @@
+# Default code snippet language
+
+Code snippets without defined programming language displays some symbols incorrectly, e.g.: `<`, `>`, `{`, `}`.
+
+So this plugin will define programming language for code snippets without specified language. 
+
+Default language: `shell`
```

Comment:
Should we add info about how to choose non-default language, or it is too obvious?

Reply:
Non-default is any language specified. It is part of markdown syntax.

Reply:
It is very hard (or impossible) to add markdown code snippet inside markdown document.